ext/HealthBaseOMOPCDMExt.jl

```diff
@@ -0,0 +1,58 @@
+module HealthBaseOMOPCDMExt
+
+using HealthBase
+using DataFrames
+using OMOPCommonDataModel
+
+function __init__()
+    @info "OMOP CDM extension for HealthBase has been loaded!"
+end
+
+"""
+    HealthTable(df::DataFrame; omop_cdm_version="5.4", disable_type_enforcement=false)
+
+Constructs a `HealthTable` for an OMOP CDM dataset by validating the given `DataFrame`.
+
+This constructor validates the `DataFrame` against the specified OMOP CDM version. It checks that
+column names are valid OMOP CDM fields and that their data types are compatible. It then
+attaches all available metadata from the OMOP CDM specification to the DataFrame's columns.
+
+If `disable_type_enforcement` is true, type mismatches will emit warnings instead of errors.
+
+Returns a `HealthTable` object wrapping the validated DataFrame.
+"""
```

Comment:
Small thing but don't forget to add sections for arguments, keyword arguments, returns, and an example within the docstring.